src/admin/install/kunena.install.upgrade.xml

```diff
@@ -274,7 +274,37 @@
                  versiondate="2024-05-25"
                  versionname="Internal">
             <query mode="silenterror">
-            ALTER TABLE `#__kunena_users` DROP `icq`; 
+            ALTER TABLE `#__kunena_users` 
+                DROP IF EXISTS `icq`, 
+                DROP IF EXISTS `x_social`, 
```

Comment:
@xillibit  the tables get not dropped on an upgrade. But if I try the query manually it works.